spec/lib/secure_headers/headers/cookie_spec.rb

```diff
@@ -5,37 +5,42 @@ module SecureHeaders
   describe Cookie do
     let(:raw_cookie) { "_session=thisisatest" }
 
-    it "does not tamper with cookies when unconfigured" do
-      cookie = Cookie.new(raw_cookie, {})
+    it "does not tamper with cookies when using OPT_OUT is used" do
```

Comment:
👍 for adding support for `SecureHeaders::OPT_OUT` this was a HUGE miss on my part in the initial implementation.